.claude/hooks/run-related-test.sh

```
@@ -0,0 +1,39 @@
+fi
+
+# 테스트/설정 파일은 스킵
+if echo "$FILE" | grep -qE '\.(spec|test)\.(ts|tsx)$|\.config\.|\.json$|\.md$|\.css$|\.html$'; then
+  exit 0
+fi
+
+cd "$CLAUDE_PROJECT_DIR" || exit 0
+
+# server 소스 → .spec.ts 찾기
+if echo "$FILE" | grep -q 'server/src/'; then
+  TEST_FILE=$(echo "$FILE" | sed 's/\.ts$/.spec.ts/')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)